lib/trebuchet/strategy/everyone.rb

```diff
@@ -0,0 +1,26 @@
+require 'singleton'
+# Everyone is to launch the feature to everyone
+class Trebuchet::Strategy::Everyone < Trebuchet::Strategy::Base
+  include Singleton
+
+  def initialize(options = nil)
+    # ignore options
+  end
+
+  def name
+    :everyone
+  end
+
+  def launch_at?(user, request = nil)
+    true
+  end
+
+  def needs_user?
+    false
+  end
+
+  def to_s
+    "everyone (everyone)"
```

Comment:
I think this could just be `everyone` or `launched to everyone`, the redundancy in parens seems weird to me. Same for `nobody` below.